modules/geolocation/outputanalysis.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+
+def get_best_location(array):
```

Comment:
Name the parameter something more descriptive e.g. inputLocationTupleList or something.

Move this function into the Geolocation class in geolocation.py.